pages/create-expense.js

```diff
@@ -207,24 +207,10 @@ class CreateExpensePage extends React.Component {
   onFormSubmit = async expense => {
     try {
       if (expense.payee.isInvite) {
-        const expenseDraft = pick(expense, [
-          'description',
-          'longDescription',
-          'tags',
-          'type',
-          'privateMessage',
-          'invoiceInfo',
-          'recipientNote',
-          'items',
-          'attachedFiles',
-          'payeeLocation',
-          'payoutMethod',
-        ]);
-        expenseDraft['payee'] = pick(expense.payee, ['id', 'slug', 'name', 'email', 'isInvite', 'organization']);
         const result = await this.props.draftExpenseAndInviteUser({
           variables: {
             account: { id: this.props.data.account.id },
-            expense: expenseDraft,
+            expense: prepareExpenseForSubmit(expense),
```

Comment:
I'm re-using the generic helper here as it seemed `prepareExpenseForSubmit` was already handling invitations properly; and that way new fields will be included by default.